### DISCUSSION TITLE
Seal Report Designer: how to get the login user ID into the Model SQL query for filtering?

### DISCUSSION THREAD

**Discussion by aazone123** [ariacom/Seal-Report#122] [General]
------------------------------------------------------------------
Select * FROM TableA
Where TableA.UserID like '{Seal Web portal login user ID}%'
------------------------------------------------------------------

Hi all, I am very new to use Seal Report Designer.
In Seal report designer > Models >
How can I get the mentioned aboive {Seal Web portal login user ID} , and use it as a SQL query condition for filter ?
Thanks.

**Comment by ariacom** [comment]
There are several ways to handle dynamic security, please check: https://sealreport.org/Recipes#lineDynamicSecurity

**Reply by aazone123** [reply]
Thanks for your help. Let me check it out.
